tests/system/robot/symbolPronunciationTests.py

```diff
@@ -374,7 +372,7 @@ def test_selByWord():
 				'left paren(quietly right paren) ',  # Expect: parenthesis are named
 				'quote Hello comma, ', 'Jim ', 'quote  dot. ',  # Expect: quote, comma and dot are named
 				'don tick t ',  # Expect: mid-word symbol substituted
-				'right-pointing arrow  ', 't-shirt  ',  # Expect dash symbol not to be replaced with word.
+				'right-pointing arrow  ', 't-shirt  ',
```

Comment:
This line can remain unchanged

```suggestion
				'right-pointing arrow  ', 't-shirt  ',  # Expect dash symbol not to be replaced with word.
```